### ISSUE TITLE
Icon subset request: keyboard buttons

### ISSUE THREAD

**Issue by molnarelodbalazs** [FortAwesome/Font-Awesome#10867]
Hello there!

It would be a nice feature if you could integrate *keyboard buttons* for `pc` and `mac` also.

for example Ctrl, Shift, Cmd, Esc and so on ;-)

**Comment by GNSubrahmanyam** [comment]
@macmladen Could you please provide an example?

https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons

**Comment by macmladen** [comment]
I was wondering how I got mentioned here? 😃 (anyway, I feel like I may have suggested this sometimes, somewhere)

Although there are PC and Mac keys already in Unicode like ⌘ ⎋ ⏎ ⏏ ⇥ ⇪  ⇧⌤ ⌥ ⌃ ⌫

Those are interesting in some more consistent, iconic form and also with drawn rounded rectangle (or without) which would indicate that keys should be pressed on keyboard and not some direction or behaviour.

Those are valuable sometimes in dropdown megamenus and could also be useful for instructions on keyboard use in help for tooltips on components.

It would be useful to have all 102 usual keyboard keys and then maybe some combination keys like (+ =) or (? /) because they help explaining what user should **press**, not type.

**Comment by molnarelodbalazs** [comment]
> In software documentation, it is very handy when you can call keyboard buttons with names.

## use cases
if you have to write the combination " Ctrl + Alt + Del  "
-   simply you can use the characters: Ctrl + Alt + Del
-   or use markdown with the tick mark `Ctrl` + `Alt` + `Del`
-   or use *font-awesome icons* for good text maintainability  :fa-win-key-ctrl: + :fa-win-key-alt: + :fa-win-key-del: to provide more accurate design

## example images (lower in size and in colors): 
![](https://img.clipartfest.com/2f3b8c868f4a6a9601ae86be8b576ec4_download-this-image-as-shift-key-clipart_600-189.png)

There could be a css class for making a button icon fa-buttonize:

```
<i class="fa fa-win-key-ctrl fa-buttonize"></i>
```
![](https://macdermott.org/blog/wp-content/uploads/2013/08/ctrl.png)


-   esc
-   f1 - f12
-   print sccreen, scroll lock, pause, break
-   insert, home, page up, page down, delete, end
-   num lock
-   the arrow keys
-   (0 - 9, § - ´  ...)
-   backspace, enter, shift
-   ctrl, win, alt, space, alt gr, 
-   tab, caps lock, 
-   (qwerty ...)
and so on

**Comment by macmladen** [comment]
I would not use `win` as nothing is windows specifically about that, those **are** just keys, except `:fa-win:` which would mark Windows key.

It could be `:fa-key-ctrl:` + `:fa-key-ins:` + `:fa-key-delete:` to get that.

It would be good to have most if not all keys but remember, there are a lot of national keyboards with some combinations, not to mention Chinese, Japanese, Korean, Indian, Cyrillic and many others.

However, I think **dead** or **modification** keys are of primary interest, secondary are all other special (like cursor keys, home, end) and then latin-26 and numbers.

**Comment by GNSubrahmanyam** [comment]
@macmladen  Sorry Sir, by-mistake I mentioned your name.

**Comment by macmladen** [comment]
I do not think it is specific, actually it is quite universal: from menu icons as shortcuts to help tip tools to instructions on shortcuts and general manual for whatever site or need, correct me if I am wrong.

At least, we should make the major ones like @molnarelodbalazs said:

* CTRL
* SHIFT
* ALT
* CMD (as command on Mac)
* WIN
* ALT GR
* SPACE
* TAB
* CAPS LOCK
* ESC
* Backspace

After those are included I am sure there will be positive reaction which may inspire inclusion of the other symbols as well.
